internal/cli/watch_opts.go

```diff
@@ -40,9 +44,17 @@ func (opts *WatchOpts) Watch(f Watcher) error {
 		return errors.New("no watcher provided")
 	}
 	opts.start()
+	opts.n = 2
 	for {
 		done, err := f()
-		if err != nil || done {
+		if err != nil {
+			if opts.exponentialBackoff(err) {
```

Comment:
[q] will this impact cluster deletion?

Reply:
@fmenezes This will impact watching every command.

Reply:
Assuming where @fmenezes is coming I also wanted to flag I though we discussed backingafter the first call being a success